gnome/gobject-introspection/Portfile

```diff
@@ -20,45 +20,70 @@ homepage            https://wiki.gnome.org/Projects/GObjectIntrospection
 master_sites        gnome:sources/${name}/${branch}/
 use_xz              yes
 
-checksums           rmd160  0c443e36b5860278de2bb77fa99997aabbda3513 \
-                    sha256  025b632bbd944dcf11fc50d19a0ca086b83baf92b3e34936d008180d28cdc3c8 \
-                    size    1378068
+checksums           rmd160  d9f1d8b15259ca9e77c5103b3bf79df4f7976ee2 \
+                    sha256  b1ee7ed257fdbc008702bdff0ff3e78a660e7e602efa8f211dc89b9d1e7d90a2 \
+                    size    980732
 
-depends_build       port:pkgconfig \
-                    port:autoconf-archive
+depends_build-append port:pkgconfig
 
 depends_lib         path:lib/pkgconfig/cairo.pc:cairo \
                     path:lib/pkgconfig/glib-2.0.pc:glib2 \
-                    port:libffi \
-                    port:py37-mako \
-                    port:py37-markdown
+                    port:libffi
 
 depends_run         bin:glibtool:libtool
 
-patchfiles          no-env-shebang.patch
+build.post_args-append --verbose
 
-post-patch {
-    reinplace "s|libcairo-gobject.2.dylib|${prefix}/lib/libcairo-gobject.2.dylib|g" ${worksrcpath}/configure.ac
+variant python27 conflicts python35 python36 python37 python38 description {Build using Python 2.7} {
+    configure.python        ${prefix}/bin/python2.7
+    depends_lib-append      port:py27-mako \
+                            port:py27-markdown
```

Comment:
Is there really a need for a variant for an unsupported version of python? I think it should be removed.

Reply:
I personally would keep Python27 because many developers who write software on gobject won't update and often macports users who would install this might just be attempting to run or package those.

I think it makes sense to keep nominal Python27 support as long as Python27 is still a port